mailstore-connector-demo/config/variables.yaml

```diff
@@ -26,3 +26,27 @@ Variables:
       properties:
           mail.imaps.ssl.checkserveridentity: false
           mail.imaps.ssl.trust: '*'
+          # only set below credential when you go with oauth2
+          # mail.imaps.auth.mechanisms: 'XOAUTH2'
+          # mail.imaps.sasl.enable: 'true'
+          # mail.imaps.sasl.mechanisms: 'XOAUTH2'
+      
+      # [basic, oauth2] basic: username and password, oauth2: currently only support OAuth2 client credentials grant flow
+      auth: 'basic'
```

Comment:
@nvvi 
It could be better when this variable is optional(when it is null, authentication='basic') 
=> file variables.yaml in case of multiple mail-connectors will be shorten for Admin users